docs/sources/alerting/_index.md

```diff
@@ -6,49 +6,47 @@ weight = 110
 
 # Alerts overview
 
-Alerts allow you to identify problems in your system moments after they occur. By quickly identifying unintended changes in your system, you can minimize disruptions to your services.
+Alerts allow you to know about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services.
 
-Alerts consists of two parts:
+Alerts have four main components:
 
-- Alert rules - When the alert is triggered. Alert rules are defined by one or more conditions that are regularly evaluated by Grafana.
-- Notification channel - How the alert is delivered. When the conditions of an alert rule are met, the Grafana notifies the channels configured for that alert.
-
-Currently only the graph panel visualization supports alerts.
+- Alert rule - One or more conditions, the frequency of evaluation, and the (optional) duration that a condition must be met before notifying.
+- Contact point - A channel for sending notifications when the conditions of an alert rule are met.
+- Notification policy - A set of matching and grouping criteria used to determine where, and how frequently, to send notifications. 
+- Silences - Date and matching criteria used to silence notifications. 
 
 ## Alert tasks
 
 You can perform the following tasks for alerts:
 
-- [Add or edit an alert notification channel]({{< relref "notifications.md" >}})
 - [Create an alert rule]({{< relref "create-alerts.md" >}})
 - [View existing alert rules and their current state]({{< relref "view-alerts.md" >}})
 - [Test alert rules and troubleshoot]({{< relref "troubleshoot-alerts.md" >}})
+- [Add or edit an alert notification channel]({{< relref "notifications.md" >}})
```

Comment:
no such thing as "notification channel" :) there are "contact points" and "notification policies"

Reply:
Thanks!